src/diffusers/models/embeddings/combined.py

```diff
@@ -0,0 +1,221 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class LabelEmbedding(nn.Module):
```

Comment:
why is this one here now?

Reply:
Because it's only used by a single class below. I followed the philosophy behind placing the attention pooling layers.

Reply:
got it! let's put it into `others` and put the attention pooling layers into `text_image`

I thought these attention pooling layers have been here for a long time and no one else has used them, so it is ok to just put them next to the class that uses it (same for `LabelEmbedding`), but if we want to come up with one rule that applies to all the same situation, I think it is better to always put them under the respective files so that it is more likely to be reused.